src/web/extractors.rs

```diff
@@ -2082,6 +2112,37 @@ mod tests {
         assert_eq!(&result.collection, "tabs");
     }
 
+    #[test]
+    fn test_quoted_id() {
+        let payload = HawkPayload::test_default(*USER_ID);
+        lazy_static! {
+            // Some libraries encode the bso id as "{guid}" which, is technically allowed
+            // since the spec says that bso ids could be any character. However, this can
+            // screw some things up, and while we've not seen occurances of this in the
+            // wild, we're going to transcode these to "normal" GUIDs.
+            static ref ALTERED_ID: String = format!("\"{{{}}}\"", *USER_ID);
+        }
```

Comment:
Why is this a static item? It looks like it can be changed to:
```suggestion
        // Some libraries encode the bso id as "{guid}" which, is technically allowed
        // since the spec says that bso ids could be any character. However, this can
        // screw some things up, and while we've not seen occurances of this in the
        // wild, we're going to transcode these to "normal" GUIDs.
        let altered_id = format!("\"{{{}}}\"", *USER_ID);
```

Reply:
I get a lifetime error when building the TestRequest later (see line 2134) `.param()` requires both key and value to be `\'static`. 

Since this is a test, and USER_ID is already a const, a `lazy_static` is a fairly simple fix.

Reply:
Hm, that is an odd requirement, but now the static makes sense.

Reply:
Alternatively, you may be able to include it in the URI instead.

Reply:
Now that the URI contains this data, we don't need to call `param` and therefore don't need to use `lazy_static`.